compiler_gym/bin/manual_env.py

```diff
@@ -43,10 +43,16 @@ def run_manual_env(env: CompilerEnv):
         # Allow the user to choose a benchmark, with the first choice being
         # to select randomly.
         benchmarks = sorted(env.benchmarks)
+        if not benchmarks:
+            print("No benchmarks available see https://facebookresearch.github.io/CompilerGym/getting_started.html#installing-benchmarks")
+            print("Exiting...")
+            env.close()
+            return
+
         # Strip default benchmark:// protocol.
         for i, benchmark in enumerate(benchmarks):
             if benchmark.startswith("benchmark://"):
-                benchmarks[i] = benchmark[len("benchmark://") :]
+                benchmarks[i] = benchmark[len("benchmark://"):]
```

Comment:
```suggestion
                benchmarks[i] = benchmark[len("benchmark://") :]
```

Out of curiosity, was it a code formatter that changed this? CompilerGym uses pre-commit to run a bunch of code formatters automatically for you on commit, see here for installation instructions:

https://github.com/facebookresearch/CompilerGym/blob/development/.pre-commit-config.yaml#L4-L30